api/resource_apps.go

```diff
@@ -95,6 +96,30 @@ func (client *Client) GetApp(ctx context.Context, appName string) (*App, error)
 					repository
 					version
 				}
+				machines{
```

Comment:
Appending this to the app query may end up being unwieldy compared to directly querying machines. But I think it's OK for now until we have a better strategy for GraphQL.